includes/fbbot/class-omise-fbbot-woocommerce.php

```diff
@@ -0,0 +1,81 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if (  class_exists( 'Omise_FBBot_WooCommerce') ) {
+  return;
+}
+
+class Omise_FBBot_WooCommerce {
+	public static function check_order_status( $order_id ) {
+	// Order status in woocommerce
+    // pending, processing, on-hold, completed, cancelled, refunded, failed
+
+    $order = wc_get_order( $order_id );
+
+    if (! $order ) {
+    	return NULL;
+    }
+
+    $status = $order->get_status();
+
+    return $status;
+	}
+
+	public static function update_order_status( $order_id, $charge ) {
+		$order = new WC_Order( $order_id );
+
+		switch ( $charge->status ) {
+			case 'failed':
+				$order->update_status( 'failed' );
+				break;
+			
+			case 'pending':
+				$order->update_status( 'on-hold' );
+				break;
+
+			case 'reversed':
+				$order->update_status( 'refunded' );
+				break;
+
+			case 'successful':
+				$order->update_status( 'processing' );
+				break;
+
+			default:
+				$order->update_status( 'on-hold' );
+				break;
+		}
+		
+		if ( $charge->paid ) {
+			$order->update_status( 'completed' );
+		}
+	}
+
+	public static function create_order( $params ) {
+		$product_id = $params['product_id'];
+		$messenger_id = $params['messenger_id'];
+		$customer_email = $params['customer_email'];
+
+    	$user = Omise_FBBot_User_Service::get_user( $messenger_id );
```

Comment:
@igroomgrim didn't you already pass this param through https://github.com/omise/omise-woocommerce/pull/58/files#diff-b0e25579a2040d94b4d78f2b56776d79R136?
Why still need to retrieve it here aa?